src/components/Personalization/flicker/index.js

```diff
@@ -29,7 +29,10 @@ export const hideElements = prehidingSelector => {
     return;
   }
 
-  const attrs = {};
+  const nonce = getNonce();
+  const attrs = {
+    ...(nonce && { nonce })
```

Comment:
Will this work in IE 11?

Reply:
![image](https://user-images.githubusercontent.com/2584844/95504289-6d96a900-0961-11eb-8ac4-742d0d6aac84.png)
Ah yes, IE11 I remember that one...

Reply:
I verified that IE is not breaking.